test/docker-compose-apicurio.yml

```diff
@@ -0,0 +1,33 @@
+# docker compose -f test/docker-compose-apicurio.yml up --quiet-pull -d --wait
+# docker compose -f test/docker-compose-apicurio.yml down
+version: "3"
+
+services:
+  registry:
+    image: apicurio/apicurio-registry:latest-snapshot
```

Comment:
Btw, we already have a compose file with the confluent schema registry that we can use in tests.

Reply:
Right. As I mentioned above, I wasn't familiar with that compatibility. In that case, it definitely makes sense to rely on that one instead. I'll remove. Thanks for the callout.